hyperspy/learn/mva.py

```diff
@@ -154,8 +156,8 @@ def decomposition(self,
         Returns
         -------
         (X, E) : (numpy array, numpy array)
-            If 'algorithm' == 'RPCA_GoDec' or 'ORPCA' and 'return_info' is True,
-            returns the low-rank (X) and sparse (E) matrices from robust PCA.
+            If 'algorithm' == 'RPCA_GoDec', 'ORPCA', 'ORNMF' and 'return_info' is True,
```

Comment:
Wondering if it would be sensible to have `'RPCA'` instead of `'RPCA_GoDec'`. To be consistent with `ORPCA` and `ORNMF`, `GoDec` should be a method and `RPCA` an algorithm? At the moment, there is only one RPCA method but they could be more in the future!

This should be considered in the context of https://github.com/hyperspy/hyperspy/issues/1678#issuecomment-312683352 too... If we want to try to more the arguments of the `blind_source_separation` and the `decomposition` method, we can keep it as it is and leave it for a separated PR and for 2.0.

Reply:
Makes sense to me, but perhaps should be addressed separately to this PR?

Reply:
Agree.

Reply:
Great - I'll keep it in mind for the near future